app/components/message_draft_body_component.html.erb

```diff
@@ -47,11 +47,20 @@
       <%= render MessageTemplateComponent.new(message: @message, is_last: @is_last) %>
     <% else %>
       <% if @message.metadata['validation_errors']&.any? %>
-        <div class="bg-red-100 border border-red-400 text-red-700 px-4 py-3 rounded relative w-full" role="alert">
-          <% @message.metadata['validation_errors'].each do |error| %>
-            <strong class="font-bold"><%= simple_format(error) %></strong>
-          <% end %>
-        </div>
+        <% if @message.metadata['validation_errors']['result'] == 'FAIL' %>
+          <div class="bg-red-100 border border-red-400 text-red-700 px-4 py-3 rounded relative w-full" role="alert">
+            <% @message.metadata['validation_errors']['message'].each do |error| %>
+              <strong class="font-bold"><%= simple_format(error) %></strong>
+            <% end %>
+          </div>
+        <% elsif @message.metadata['validation_errors']['result'] == 'WARN' %>
+          <div class="bg-orange-100 border border-orange-500 text-orange-700 px-4 py-3 rounded relative w-full" role="alert">
+            <strong>Upozornenie:</strong>
+            <% @message.metadata['validation_errors']['message'].each do |error| %>
+              <p><%= simple_format(error) %></p>
+            <% end %>
+          </div>
+        <% end %>
```

Comment:
Toto sa mi mierne nezda. Co ak mas chyby aj warn aj fail?

Reply:
Skusala som, FS API to v takom pripade vracia cele ako 'FAIL'. Lepsie by to bolo asi rozdelit, to asi vieme spravit, @celuchmarek?

Reply:
Ja tam vraciam celkový výsledok, čo je v tom prípade fail. Ale prípadne viem v tom texte vrátiť html aj s tagmi a v nich je potom iná classa pre warn a fail. Môže byť?

Reply:
@jsuchal chceme to delit na WARN a FAIL alebo nechame iba celkovy vysledok?

Reply:
@celuchmarek imho by sme mali rozlisovat tie priznaky na urovni error hlasky. To, ze je to "celkovo" warn/fail by som nechal ako je.

Reply:
Spravis upravu na strane FS API @celuchmarek?

Reply:
@luciajanikova jo, to už čaká na Jana: https://github.com/slovensko-digital/fs-api/pull/30  Ale môžeš aj ty pozrieť, ako som upravil response, ak chceš.

Reply:
@luciajanikova mergnuté a nasadené na staging aj prod. V GO sa toho asi nemám ja chytať, že?

Reply:
V GO upravim, vdaka!